doc/gh-pages.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+cd ..
+SOURCES="doc examples"
+git checkout gh-pages
+rm -rf _build _sources _static
```

Comment:
Can you give me push access to your repo? I'll push directly to your branch
and make the modifications to the makefile

On Sunday, 15 May 2016, Titipat Achakulvisut <notifications@github.com>
wrote:

> In doc/gh-pages.sh
> <https://github.com/pavanramkumar/pyglmnet/pull/83#discussion_r63289022>:
>
> > @@ -0,0 +1,16 @@
> > +#!/bin/bash
> > +cd ..
> > +SOURCES="doc examples"
> > +git checkout gh-pages
> > +rm -rf _build _sources _static
>
> It would be great if you can modify this file tho. It would be helpful for
> us to run a script to automatically update gh-pages (before we move to
> circle-ci)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/pavanramkumar/pyglmnet/pull/83/files/5a0262546b9da802963f53d2950d17b7c31db70a#r63289022>
>